.github/workflows/ci.yml

```diff
@@ -382,9 +382,7 @@ jobs:
         shell: bash
         working-directory: src/github.com/containerd/containerd
         run: |
-          # TODO: when https://github.com/containerd/containerd/pull/8691 makes it into the next release (container v1.6.22?), remove the skip
-          # `-skip` is only available in go1.20
-          export EXTRA_TESTFLAGS='-timeout=20m -run="[^(TestConvert)]"'
+          export EXTRA_TESTFLAGS='-timeout=20m -run="[^(TestContainerSymlinkVolumes)]"'
```

Comment:
LOL, I was about to leave a comment on the previous commit that this could be removed; generally I'd recommend making changes on `main` first, and backport them to release branches, as it's easy to forget "forward-porting" changes to `main`, in which case other release branches may miss fixes.

I was already preparing a quick PR to update `main` as well; will open a PR shortly.

Reply:
- Opened https://github.com/microsoft/hcsshim/pull/1910

Reply:
Thanks for looking into this!